src/Tribe/Admin_List.php

```diff
@@ -181,16 +181,22 @@ public static function filter_by_aggregator_record( array $clauses, WP_Query $wp
 			}
 
 			global $wpdb;
-			
+
+            $child_args = array(
+                'post_type'   => TEC_AR::$post_type,
+                'post_parent' => $record_num,
+            );
+            $child_ids = implode( ",", array_keys( get_children( $child_args, ARRAY_A ) ) );
```

Comment:
I think you can avoid this one by updating the where clause instead.

Reply:
_See example on `where` section_

Reply:
We're not looking at the parent for the event, we're looking at the aggregator record ID in the meta of the event. The aggregator record's parent is the integer that gets passed into the page. We have to pull the children IDs separately and search for any events that have those IDs in their meta. The event's parent doesn't play a part in this.

Reply:
Oh right, makes sense then.

What's the value or where is located `TEC_AR` I'm not able to see it on the file.

Reply:
Now, THAT would be a mistake on my part. Good catch.

Reply:
Great usually that one is called `Records` I think would be ideal to keep consistent with other files like this one: 

- https://github.com/moderntribe/the-events-calendar/blob/d1902db0aef3f14a2503410f3fdb5e86bd089add/src/Tribe/Aggregator/Cron.php#L5